README.md

```diff
@@ -1 +1,197 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#LICENSE)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome-books <a name="about-project"></a>
+
+
+**Awesome-books** is a website that displays my Techinical skills and projects that we have done.
+
+
+**Link to online version of Awesome-books** is in [Live Demo](#live-demo) Coming-soon
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JAVASCRIPT">JAVA-SCRIPT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **local-storage**
+- **Add-books**
+- **Remove-books**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](Coming-Soon) Coming-soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Modern chrome Browser
+
+### Setup
+
+- Clone this repository to your desired folder with the following commend
+```
+git clone git@github.com:migo1/awesomeBooks/create-acess-book-object-js.git
+```
+
+
+### Install
+
+- Type cd 
+```
+Awesome-books
+```
+- open index.html file in the browser
+
+### Usage
+
+- To run the project, execute the following command:
+  - Type cd ```awesomeBooks ```
+  - open index.html file in the browser
+
+### Run tests
+
+- To run tests, run the following command:
+  - N/A
+
+### Deployment
+
+- You can deploy this project using:
+  - Type cd ```awesomeBooks ```
+  - open index.html file in the browser
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ansar Ibrahim**
+
+- GitHub: [Ansar Ibrahim](https://github.com/AnsarIbrahim)
+- Twitter: [Ansar Ibrahim](https://twitter.com/ansaradheeb)
+- LinkedIn: [Ansar Ibrahim](https://linkedin.com/in/ansar-ibrahim-61447424a/)
+
+👤 **Robert Migot**
+
+- GitHub: (https://github.com/migo1)
+- Twitter: (https://twitter.com/@ert49554392)
+- LinkedIn: (https://www.linkedin.com/in/robert-m-3b3b1116a/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Ruby**
+- [ ] **React**
+
```

Comment:
- Hey, Adding react and ruby is beyond the scope of this project. please, check the project requirement for the bigger picture to update this. You would find other milestones, so you can use that to update the Future Features.

![image](https://github.com/migo1/awesomeBooks/assets/59397159/2784da4f-fcba-4801-acc7-98c5a5e766e0)